### ISSUE TITLE
I HAVE A QUESTION AT _PASSWORD = os.environ.get("IG_PASSWORD", "password*")

### ISSUE THREAD

**Issue by apambalik1337** [adw0rd/instagrapi#1373]
Hey when i want to enter my account password at the _PASSWORD = os.environ.get("IG_PASSWORD", "password*") . Does i need to remove the sysmbol * in  "password*") ?

**Comment by blank2121** [comment]
you need to remove the entire second argument of "password*". ```os.environ``` is treated as a dict so if you story your environment var as 
```
{
"IG_PASSWORD": "mySecurePassword"
}
```
then to get that password you would use

```python
_PASSWORD = os.environ.get("IG_PASSWORD")
# print(_PASSWORD)
# prints 'mySecurePassoword'

**Comment by apambalik1337** [comment]
so i need to change IG_PASSWORD with my password? so if i ask a stupid question im newbie

**Comment by blank2121** [comment]
My mistake, I thought you wanted to get your password from the environment variable. if you are using this for a personal project and you are not worried your password being in your local project, you can just write ```_PASSWORD = "yourpassword"```.

if you want to set you environment var in code however use the following:

```python
>>> os.environ['IG_PASSWORD'] = "yourpass"     # Set a new env. variable (String value)
```

also the reason that there are two arguments here: ```_PASSWORD = os.environ.get("IG_PASSWORD", "password*")``` is because if the environ variable ```IG_PASSWORD``` does not exist, it will return ```password*```.

**Comment by blank2121** [comment]
It depends, is this issue your own fork/project? Or is this issue under someone else's project? If the latter, then I'd say no but if this is your project, then ya it's fine.

**Comment by apambalik1337** [comment]
> It depends, is this issue your own fork/project? Or is this issue under someone else's project? If the latter, then I'd say no but if this is your project, then ya it's fine.

I dont have issue or with someonelse. Just me haha

**Comment by blank2121** [comment]
No, I mean when you make a question like you, did, you have to make it on an issue on a repository. This thread we're on right now is called an issue. Did you create that issue thread on your own repository or someone else's

**Comment by apambalik1337** [comment]
Not my own repo, sorry because of my english I can't understand you very well. I will close this issue
